cmd/otel-agent/main.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/DataDog/datadog-agent/cmd/internal/runcmd"
 	"github.com/DataDog/datadog-agent/cmd/otel-agent/command"
 	"github.com/DataDog/datadog-agent/pkg/util/flavor"
+	_ "github.com/DataDog/datadog-agent/pkg/version"
```

Comment:
Because we need the globals that are then set at build time with `-X` build directives. Even though we are using `pkg/version` elsewhere, having it here in the `main` ensures we will never break internal versioning or the build if it's removed from `comp/otelcol` or any other reference.